Workbooks/Storage/Overview/Overview.workbook

```diff
@@ -174,19 +177,82 @@
             }
           }
         ],
-        "style": "above",
+        "style": "pills",
         "queryType": 1,
         "resourceType": "microsoft.resourcegraph/resources"
       },
       "conditionalVisibility": null,
       "name": "parameters - 1"
     },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "where type == 'microsoft.storage/storageaccounts' |summarize count() by type ",
+        "size": 3,
+        "queryType": 1,
+        "resourceType": "microsoft.resourcegraph/resources",
+        "crossComponentResources": [
+          "{Subscription}"
+        ],
+        "visualization": "tiles",
+        "tileSettings": {
+          "titleContent": {
+            "columnMatch": "type",
+            "formatter": 16,
+            "formatOptions": {
+              "showIcon": true
+            }
+          },
+          "leftContent": {
+            "columnMatch": "count_",
+            "formatter": 12,
+            "formatOptions": {
+              "palette": "auto",
+              "showIcon": true
+            },
+            "numberFormat": {
+              "unit": 17,
+              "options": {
+                "maximumSignificantDigits": 3,
+                "maximumFractionDigits": 2
+              }
+            }
+          },
+          "showBorder": false
+        }
+      },
+      "conditionalVisibility": null,
+      "customWidth": "20",
+      "name": "query - 20"
+    },
+    {
+      "type": 3,
```

Comment:
if this health stuff isn't ready, just remove it?  (right now it's still running)

Reply:
fixed in new PR